RevenueCatUI/PaywallView.swift

```diff
@@ -254,6 +255,23 @@ public struct PaywallView: View {
 
         #if PAYWALL_COMPONENTS
         if let paywallComponents = offering.paywallComponents {
+            // For fallback view
+            let paywall: PaywallData = .createDefault(with: offering.availablePackages,
+                                                      locale: self.locale)
+            let paywallView = LoadedOfferingPaywallView(
+                offering: offering,
+                activelySubscribedProductIdentifiers: activelySubscribedProductIdentifiers,
+                paywall: paywall,
+                template: PaywallData.defaultTemplate,
+                mode: self.mode,
+                fonts: fonts,
+                displayCloseButton: self.displayCloseButton,
+                introEligibility: checker,
+                purchaseHandler: purchaseHandler,
+                locale: self.locale,
+                showZeroDecimalPlacePrices: showZeroDecimalPlacePrices
+            )
+
```

Comment:
This feels gross but we'll replace this eventually with a custom paywalls v2 fallback